### ISSUE TITLE
ScheduledExecutorServiceTest.scheduleWithRepetition

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#9493]
```
java.lang.AssertionError: expected:<0> but was:<1>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceTest.scheduleWithRepetition(ScheduledExecutorServiceTest.java:601)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK6/com.hazelcast$hazelcast/421/testReport/junit/com.hazelcast.scheduledexecutor/ScheduledExecutorServiceTest/scheduleWithRepetition/

**Comment by tkountis** [comment]
Suggested fix #9580

**Comment by tkountis** [comment]
#9580 merged.
